### PR TITLE
Pythran update: __builtin__ -> builtins

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

**Comment by serge-sans-paille** [comment]
@scoder can you consider merging this one? This PR is required to keep cython and pythran in sync.

File: Cython/Compiler/Pythran.py [hunk 1]
```diff
@@ -154,7 +163,7 @@ def to_pythran(op, ptype=None):
     if is_type(op_type, ["is_pythran_expr", "is_numeric", "is_float", "is_complex"]):
         return op.result()
     if op.is_none:
-        return "pythonic::__builtin__::None"
+        return "pythonic::{}::None".format(pythran_builtins)
```
Comment (scoder) [inline Cython/Compiler/Pythran.py]: Since Cython uses `%` formatting everywhere for code generation, please also stick to it here.

Comment (serge-sans-paille) [reply Cython/Compiler/Pythran.py]: Done!

**Comment by serge-sans-paille** [comment]
Thanks for the review o/

**Comment by serge-sans-paille** [comment]
And yeah it could go to 0.29.x

**Comment by serge-sans-paille** [comment]
thanks!

**Pull request merged**